examples/connection_pools/Rocket.toml

```diff
@@ -0,0 +1,14 @@
+[[global.databases]]
+name = "my_sqlite_db"
+url = "db.sqlite"
+pool_size = 10
+
+[[global.databases]]
+name = "my_primary_database"
+url = "postgres://postgres@localhost/primary_db"
+pool_size = 10
+
+ [[global.databases]]
```

Comment:
Why the indent?

Reply:
Looks like a typo. Thanks for catching it.